charts/cluster/templates/cluster.yaml

```diff
@@ -57,6 +57,12 @@ spec:
       {{- toYaml . | nindent 6 }}
     {{ end }}
 
+  managed:
```

Comment:
Can you move the line above so the managed section is not included empty?

Reply:
Sure I can do that, I just wonder if you have a specific disagreement with the rationale I provided for having the `{{ with ... }}` inside `managed`? 

If at some point there are additional elements in `managed` that you want to be able to express in the chart, you would have to do something like that to avoid the empty `managed`:

```
  {{- if or (.Values.cluster.roles) (.Values.cluster.someNewStuff) }}
  managed:
    {{- with .Values.cluster.roles }}
    roles:
      {{- toYaml . | nindent 6 }}
    {{ end }}
    {{- with .Values.cluster.someNewStuff }}
    roles:
      {{- toYaml . | nindent 6 }}
    {{ end }}
  {{ end }}
```

And AFAICT, having the empty `managed` item has no effect.

Reply:
Hi, any news on this? If you have a strong opinion on why to avoid rendering the empty `managed` element I can do as you suggest, I would just like to understand the rationale given the argument I provided.

Reply:
Your argument makes sense.